core/src/main/java/org/apache/carbondata/core/datamap/DataMapStoreManager.java

```diff
@@ -172,6 +174,48 @@ public void dropDataMapSchema(String dataMapName) throws IOException {
     provider.dropSchema(dataMapName);
   }
 
+  /**
+   * Update the datamap schema to storage by table rename
```

Comment:
shouldn't be `Update the datamap schema after table rename`?

Reply:
ok，i am modifying it.